CONTRIBUTING.md

```diff
@@ -131,6 +131,26 @@ option for this.
 
 4. To Deploy, simply push your changes to the `canary` branch, then visit "https://search-ui-canary.netlify.com/"
 
+### Release candidates
+
+When pushing release candidates, the following lerna commands can be useful:
+
+```
+# Create a pre-release version, like 1.0.0-rc.0
+npx lerna version prerelease --exact --preid rc
+# Publish 1.0.0-rc.0 and update the `next` to point to this version
+npx lerna publish from-package --force-publish=* --dist-tag next
+```
+
+### Testing Canary build and Release Pre-releases
+
+We have a number of demos available that you can use to do quick smoke testing of releases in various
+stacks:
+
+https://codesandbox.io/s/search-ui-gatsby-example-u041m
+https://codesandbox.io/s/search-ui-next-js-example-tb05u
+https://codesandbox.io/s/search-ui-national-parks-example-kdyms
```

Comment:
I think this needs to have `- ` bullet point prefixes - right now it's just outputting a single line of links in a row which is a bit hard to click.

<img width="872" alt="" src="https://user-images.githubusercontent.com/549407/61251996-fac46100-a710-11e9-9798-6980f93c0d36.png">

The stable demos section/links below would probably benefit from the same.

Reply:
Oh yes, you're right. Thank you :)